modules/mesh-task/main.tf

```diff
@@ -54,14 +54,136 @@ locals {
   upstreams_flag = join(",", [for upstream in var.upstreams : "${upstream["destination_name"]}:${upstream["local_bind_port"]}"])
 }
 
+resource "aws_iam_role" "task" {
+  name = "${var.family}-task"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Principal = {
+          Service = "ecs-tasks.amazonaws.com"
+        }
+      },
+    ]
+  })
+
+  inline_policy {
+    name   = "exec"
+    policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+%{if local.dev_server_enabled~}
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ecs:ListTasks",
+        "ecs:DescribeTasks"
+      ],
+      "Resource": [
+        "*"
+      ]
+    },
```

Comment:
I believe this is is only needed if dev server is enabled for the `discover-servers` command.

Reply:
correct